exonum-light-client/pom.xml

```diff
@@ -118,6 +119,13 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+      <version>${protobuf.version}</version>
+      <scope>provided</scope>
```

Comment:
Why this dependency is needed? Doesn't common bring it as a transitive? 
It _may_ be specified explicitly, but certainly not as `provided`. If it isn't already available — please fix the common.

Reply:
It is explicitly marked as an optional dependency and is not passed on transitively - https://github.com/exonum/exonum-java-binding/blob/master/exonum-java-binding/common/pom.xml#L47

Reply:
Added a PR that makes protobuf dependency transitive - https://github.com/exonum/exonum-java-binding/pull/1246